### ISSUE TITLE
 Dropdown - In Scan Mode, Narrator Focus moves to “dropdown” twice

### ISSUE THREAD

**Issue by sagaranandani17** [OfficeDev/office-ui-fabric-react#11129]
### Environment Information

- **Package version(s)**: 7.24.0
- **OS version:** 1903 (OS Build: 18326.295)
- **Browser:** Edge(Microsoft Edge 44.18362.267.0)

**User Experience:** 
If in scan mode, narrator focus moves twice on controls then user will face difficulties while navigating on the control as they expect a single focus on each control.

**Steps to reproduce**
1) Go to office fabric link(https://developer.microsoft.com/en-us/fabric#/controls/web/dropdown)
2) In scan mode on, bring focus on Basic dropdown(Basic uncontrolled example) label,  Narrator will say-
**Basic uncontrolled example**
Press down arrow key
**Apple selected , selection contains 7 items**
Press down arrow key again
**one of seven selected apple**

Here, we have seen that focus is coming twice on dropdown in scan mode

**Actual behavior:**
 In Scan Mode,Narrator Focus moves to “Dropdown” twice.

**Expected behavior:**
 In Scan Mode,Narrator Focus should not move to “dropdown” twice.

**Please find the attachment**
![image](https://user-images.githubusercontent.com/57341535/68460275-6a64db80-022d-11ea-9cab-1a8ce79e2f13.png)

for more details please have look to the video
[MAS4.2.10_Portal - Foam.zip](https://github.com/OfficeDev/office-ui-fabric-react/files/3823518/MAS4.2.10_Portal.-.Foam.zip)

**Comment by JasonGore** [comment]
Dropdown uses `listbox` role. Similar [ARIA listbox examples](https://www.w3.org/TR/2017/WD-wai-aria-practices-1.1-20170628/examples/listbox/listbox.html) behave the same way when entered in scan mode: 

> Important Features: <mark name="ContinuousReadingEnd Id:1"/>

Down arrow
> Important Features:, selected, selection contains 10 items, <mark name="ContinuousReadingEnd Id:2"/>

Down arrow
> <say-as interpret-as="spell">1</say-as> of 10, Proximity of public K-12 schools <mark name="ContinuousReadingEnd Id:3"/>


This appears to be Narrator behavior and should be logged against Narrator if considered a defect.
